web/src/engine/websites/YugenMangasPT_e2e.ts

```diff
@@ -6,18 +6,18 @@ const config = {
         title: 'YugenMangas (PT)'
     },
     container: {
-        url: 'https://yugenmangas.org/series/para-amar-seu-inimigo/',
-        id: '/series/para-amar-seu-inimigo/',
+        url: 'https://yugenmangas.net.br/series/para-amar-seu-inimigo',
+        id: 'para-amar-seu-inimigo',
```

Comment:
Change of ID seems to be a breaking change, will bookmarks with previous ID still work?

Reply:
They just wont anyway 🤷  . Reason is YugenMangas keeps changing everything : API, domains, paths, rebranding... IIRC before that it WASNT an api :/. Previous bookmark would be broken without us, because of yugenmangas itself.

First i was planning to use https://ikigaimangas.com.br/ as website domain, since https://yugenmangas.org/ redirect to it but its using 'https://yugenmangas.net.br/ to store pictures and for the api domain.

There is another way tho : use their rebranded name for both YugenMangas 
https://ikigaimangas.com.br/ and https://ikigaimangas.com/ (old yugenmangas.lat)

Bookmark migration wont work anyway i would have coded it otherwise.